samcli/commands/init/__init__.py

```diff
@@ -77,13 +77,34 @@ def do_cli(ctx, location, runtime, output_dir, name, no_input):
     LOG.debug("Init command")
     click.secho("[+] Initializing project structure...", fg="green")
 
+    no_build_msg = """
+Project generated: {output_dir}/{name}
+
+Steps you can take next within the project folder
+===================================================
+[*] Invoke Function: sam local invoke HelloWorldFunction --event event.json
+[*] Start API Gateway locally: sam local start-api
+""".format(output_dir=output_dir, name=name)
+
+    build_msg = """
+Project generated: {output_dir}/{name}
+
+Steps you can take next within the project folder
+===================================================
+[*] Install dependencies
+[*] Invoke Function: sam local invoke HelloWorldFunction --event event.json
+[*] Start API Gateway locally: sam local start-api
+""".format(output_dir=output_dir, name=name)
+
+    no_build_step_required = (
+        "python", "python3.7", "python3.6", "python2.7", "nodejs", "nodejs4.3", "nodejs6.10", "nodejs8.10", "ruby2.5")
+    next_step_msg = no_build_msg if runtime in no_build_step_required else build_msg
```

Comment:
Not sure what you mean by a custom template takes in 'a in the --runtime arg' -- This is a quick workaround until sam build fully supports all runtimes so the message will remain consistent and this will be deleted, though the workaround is flawed in the sense of as `sam build` supports more native runtimes this will need updating. 

```python
    no_build_step_required = (
        "python", "python3.7", "python3.6", "python2.7", "nodejs", "nodejs4.3", "nodejs6.10", "nodejs8.10", "ruby2.5")
    next_step_msg = no_build_msg if runtime in no_build_step_required else build_msg
```

``--runtime`` arg is [validated against the ones we support](https://github.com/awslabs/aws-sam-cli/blob/develop/samcli/commands/init/__init__.py#L20) and `no_build_step_required` essentially borrows the ones that have sam build support.

What would you suggest instead?

Reply:
I see. I was getting two PRs mixed up, where I thought we were passing runtime to cookiecutter examples that are passed to `--location`. You can ignore this comment and can handle in the other pr that enables that.